testing/run_tests.py

```diff
@@ -282,13 +284,47 @@ def EnsureJavaTestsAreBuilt(android_out_dir):
   RunCmd(gn_command, cwd=buildroot_dir)
   RunCmd(ninja_command, cwd=buildroot_dir)
 
+def EnsureIosTestsAreBuilt(ios_out_dir):
+  ninja_command = [
+    'autoninja',
+    '-C',
+    ios_out_dir,
+    'ios_test_flutter'
+  ]
+
+  # Attempt running Ninja if the out directory exists.
+  # We don't want to blow away any custom GN args the caller may have already set.
+  if os.path.exists(ios_out_dir):
+    RunCmd(ninja_command, cwd=buildroot_dir)
+    return
+
+  assert ios_out_dir != "out/ios_debug_sim_unopt", "%s doesn't exist. Run GN to generate the directory first" % ios_out_dir
+
+  # Otherwise prepare the directory first, then build the test.
+  gn_command = [
+    os.path.join(buildroot_dir, 'flutter', 'tools', 'gn'),
+    '--ios',
+    '--unoptimized',
+    '--runtime-mode=debug',
+    '--no-lto',
+    '--simulator'
+  ]
+  RunCmd(gn_command, cwd=buildroot_dir)
+  RunCmd(ninja_command, cwd=buildroot_dir)
+
 def AssertExpectedJavaVersion():
   EXPECTED_VERSION = '1.8'
   # `java -version` is output to stderr. https://bugs.java.com/bugdatabase/view_bug.do?bug_id=4380614
   version_output = subprocess.check_output(['java', '-version'], stderr=subprocess.STDOUT)
   match = bool(re.compile('version "%s' % EXPECTED_VERSION).search(version_output))
   message = "JUnit tests need to be run with Java %s. Check the `java -version` on your PATH." % EXPECTED_VERSION
-  assert True, message
+  assert match, message
+
+def AssertExpectedXcodeVersion():
+  EXPECTED_MAJOR_VERSION = '11'
```

Comment:
This is probably better suited as an arg to this function, no?

Nit: return a bool and assert at a higher level

Reply:
This file's just a rather ad-hoc script. I doubt we'll ever touch this line again.